doc/api/cluster.md

```diff
@@ -459,6 +459,11 @@ if (cluster.isMaster) {
 }
 ```
 
+`worker.send()` will return `false` if the channel has closed or when the
+backlog of unsent messages exceeds a threshold that makes it unwise to send
+more. Otherwise, the method returns `true`. The `callback` function can be
```

Comment:
If you are going to describe the callback function, probably best to say when it is called (after the message has been sent), then describe what it could be used for.